### PR TITLE
[6.x] Add missing throws docblocks

### PR DESCRIPTION
**Pull request by iamgergo**

### CODE REVIEW

File: src/Illuminate/View/View.php [hunk 1]
```diff
@@ -84,6 +84,7 @@ public function __construct(Factory $factory, Engine $engine, $view, $path, $dat
      * @return array|string
      *
      * @throws \Throwable
+     * @throws \Exception
```
Comment (GrahamCampbell) [inline src/Illuminate/View/View.php]: :-1:

File: src/Illuminate/Routing/Middleware/ThrottleRequestsWithRedis.php [hunk 2]
```diff
@@ -50,7 +50,7 @@ public function __construct(Redis $redis)
      * @param  string  $prefix
      * @return mixed
      *
-     * @throws \Symfony\Component\HttpKernel\Exception\HttpException
+     * @throws \Illuminate\Http\Exceptions\ThrottleRequestsException
```
Comment (GrahamCampbell) [inline src/Illuminate/Routing/Middleware/ThrottleRequestsWithRedis.php]: :-1:

File: src/Illuminate/Foundation/Testing/TestCase.php [hunk 3]
```diff
@@ -142,6 +142,9 @@ protected function setUpTraits()
      * Clean up the testing environment before the next test.
      *
      * @return void
+     *
+     * @throws \Throwable
+     * @throws \Mockery\Exception\InvalidCountException
```
Comment (GrahamCampbell) [inline src/Illuminate/Foundation/Testing/TestCase.php]: :-1:

File: src/Illuminate/Foundation/Testing/Concerns/InteractsWithExceptionHandling.php [hunk 4]
```diff
@@ -113,6 +113,7 @@ public function shouldReport(Exception $e)
              * @return \Symfony\Component\HttpFoundation\Response
              *
              * @throws \Exception
+             * @throws \Symfony\Component\HttpKernel\Exception\NotFoundHttpException
```
Comment (GrahamCampbell) [inline src/Illuminate/Foundation/Testing/Concerns/InteractsWithExceptionHandling.php]: :-1:

File: src/Illuminate/Database/SqlServerConnection.php [hunk 5]
```diff
@@ -20,7 +20,8 @@ class SqlServerConnection extends Connection
      * @param  int  $attempts
      * @return mixed
      *
-     * @throws \Exception|\Throwable
+     * @throws \Exception
```
Comment (GrahamCampbell) [inline src/Illuminate/Database/SqlServerConnection.php]: :-1:

**Review by GrahamCampbell — CHANGES_REQUESTED** [review]
Thanks, but a lot of these changes don't look right. Whenever a more general type is in the throw doc, specific versions of it should not be added.

**Comment by iamgergo** [comment]
Well, based on this it's just enough adding a throws tag with a `\Throwable`. This was the problem with the `mixed` return types (#31267), that they are too generic.

**Pull request closed without merging**